specifyweb/specify/build_models.py

```diff
@@ -97,6 +107,12 @@ def protect(collector, field, sub_objs, using):
     'Spappresourcedata.spappresource': models.CASCADE,
     'Spappresourcedata.spviewsetobj': models.CASCADE,
     'Spreport.appresource': models.CASCADE,
+
+    'Geographytreedefitem.parent': models.DO_NOTHING,
+    'Geologictimeperiodtreedefitem.parent': models.DO_NOTHING,
+    'Lithostrattreedefitem.parent': models.DO_NOTHING,
+    'Storagetreedefitem.parent': models.DO_NOTHING,
+    'Taxontreedefitem.parent': models.DO_NOTHING,
```

Comment:
are these necessary here? seems excessive (bc some other code could naively perform a delete and django wouldn't say no). Since it doesn't affect the database itself, this doesn't cause a problem. 

however, apparently the delete blockers do not report parent relationship bc of this (even after removing the filtering logic). Would a cleaner way be to remove these, and have filtering logic on the frontend?

Reply:
On second thought, this seems fine, bc we just handle it.